### DISCUSSION TITLE
Question about template "file_groupowner"

### DISCUSSION THREAD

**Discussion by alanmcanonical** [ComplianceAsCode/content#12162] [Q&A] [answered]
The file_groupowner bash template doesn't use "-L" option to track symbolic link which leave all the actual file unattended during remediation. I'm not sure about whether this behavior is expected or not.

**Comment by Mab879** [comment]
This also seems to be an issue with the `file_owner` template as well.

**Reply by alanmcanonical** [reply]
Yes

**Comment by Mab879** [comment]
I'm working on a pull request that make the template follow symlinks. That should be up in the coming days.

**Reply by Mab879** [reply, marked as answer]
See #12214

**Reply by alanmcanonical** [reply]
Here is my thought: All the files that symbolic point to can only be one of the two case: (a) files sit inside _path_ or (b) file sit outside _path_, since we already use `find`, (a) would already be visited. So my concern is the (b) case which I am not sure whether we should handle according to compliance requirement.
